glustercli/cmd/georep.go

```diff
@@ -158,7 +158,7 @@ func parseRemoteData(data string) (string, string, string, error) {
 	remotehostvol := strings.Split(data, "::")
```

Comment:
The commit message title should be limited to 50 characters or so. Unfortunately, Github doesn't allow me to review commit messages. Please note this in future commits and PRs you send.